lib/core/src/lib/app-config/app-config.service.spec.ts

```diff
@@ -188,4 +188,10 @@ describe('AppConfigService', () => {
 
         expect(appConfigService.get('files.excluded')[0]).toBe('excluded');
     });
+
+    it('should emit on extension load', () => {
+        appConfigService.onExtensionSetup$.subscribe((config) => {
```

Comment:
have you tested that the unit is red first?

Reply:
the test is async, so probably never reaches the "expect"

Reply:
hmm the test is not async, `it('should emit on extension load', () => {..}` no `async` there 

Nevertheless I reverted this changes, since I will have different approach after discussion with @mauriziovitale 

Just needs to update typings for [this](https://github.com/Alfresco/alfresco-apps/pull/4773#discussion_r1087739879)